UnityProject/Assets/Scripts/Electricity/NodeModules/ResistanceSourceModule.cs

```diff
@@ -60,6 +60,19 @@ public void BroadcastSetUpMessage(ElectricalNodeControl Node)
 		ElectricalSynchronisation.PoweredDevices.Add(ControllingNode);
 		Node.AddModule(this);
 	}
+	public override void ObjectStateChange(ObjectState tState) {
```

Comment:
You might want to find out what you "Format Document" shortcut is in the ide you are using so you can fix formatting easier.

Reply:
still requires formatting